release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1592,7 +1586,7 @@ sourceStrategy:
 [id="ocp-4-11-machine-config-operator-bug-fixes"]
 ==== Machine Config Operator
 
-* Because `keyFile` is not configured as the default plug-in for NetworkManager on RHEL nodes, RHEL nodes might not reach the ready state after a reboot.  With this fix, `keyFile` is set as the default NetworkManager plug-in on all cluster nodes. As a result, nodes correctly reach the ready state after a reboot. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060133[*BZ#2060133*])
+* Because `keyFile` is not configured as the default plug-in for NetworkManager on RHEL nodes, RHEL nodes might not reach the ready state after a reboot. With this fix, `keyFile` is set as the default NetworkManager plug-in on all cluster nodes. As a result, nodes correctly reach the ready state after a reboot. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060133[*BZ#2060133*])
```

Comment:
```suggestion
* Because `keyFile` is not configured as the default plug-in for NetworkManager on RHEL nodes, RHEL nodes might not reach the ready state after a reboot. With this fix, `keyFile` is set as the default NetworkManager plug-in on all cluster nodes. As a result, nodes correctly reach the `ready` state after a reboot. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060133[*BZ#2060133*])
```